src/exo/API_scheduling.py

```diff
@@ -890,6 +997,10 @@ def commute_expr(proc, expr_cursors):
 
     args:
         expr_cursors - a list of cursors to the binary operation
+        rc           - (bool) whether to return relevant cursors
+
+    relevant cursors:
+        exprs        - a tuple of cursors to the commuted expressions
```

Comment:
This seems fine, but dropping comment because I hadn't seen this scheduling op before.

Should it be called `commute_exprs` if it takes a list of cursors?  Also, should we just stick to the convention that it should take a single cursor, but could be applied repeatedly?

Reply:
Same comment for similar expr ops added after I last went through this file